dashboard/test/ui/features/platform/policy_compliance.feature

```diff
@@ -264,3 +264,27 @@ Feature: Policy Compliance and Parental Permission
 
     # And it should tell me that the request was granted
     And element "#permission-status" contains text "Granted"
+  
+  Scenario: Student in the CPA flow cannot change their age or state
+    Given I create an authorized teacher-associated under-13 student in Colorado named "Tandy" after CPA exception
```

Comment:
Some other test cases to consider:

* Over-13 and in Colorado?\
* U13, Colorado, Clever-only

Reply:
Ill add these

Reply:
What is the distinction between Clever users and LTI users that makes us want to lock the state and age dropdowns for LTI but not for Clever?

Reply:
> What is the distinction between Clever users and LTI users that makes us want to lock the state and age dropdowns for LTI but not for Clever?

I don't think we want to lock it for LTI either. Clever and LTI should be treated the same.